doc/API/centreon-api-v2.yaml

```diff
@@ -4543,3 +4607,45 @@ components:
             - To link a remote server to a Central
             - To link a poller to a remote server
           example: "1.1.1.1"
+    Gorgone.Response:
+      type: object
+      properties:
+        message:
+          type: string
+          description: "Message returned by Gorgone"
+          example: "Logs found"
+        token:
+          type: string
+          description: "Token used to identify the command"
+        data:
+          type: array
+          items:
+            type: object
+            properties:
+              creation_time:
+                type: string
+                format: date-time
+                description: "Creation of dataset on a ISO Format"
+                example: "2020-11-24T09:13:04+01:00"
+              event_time:
+                type: string
+                format: date-time
+                description: "Time when event occured on a ISO Format"
+                example: "2020-11-24T09:11:00+01:00"
+              id:
+                type: number
+                format: integer
+                description: "Id of the Event"
+                example: 1
+              token:
+                type: string
+                description: "Token used to identify the command"
+              code:
+                type: number
+                format: integer
+                description: "Return code of response"
+                example: 2
+              data:
+                type: string
+                description: "Data on JSON Format"
+                example: "{\"action\":\"getthumbprint\",\"data\":{\"thumbprint\":\"xxxx_xxxxxxxxx\"},\"message\":\"ok\"}"
```

Comment:
Response is a stringified json ?
If not, type is a object and define properties

Reply:
Yes it is a stringified JSON